src/status_im/acquisition/core.cljs

```diff
@@ -93,3 +100,8 @@
   {}
   [_]
   {::check-referrer nil})
+
+(re-frame/reg-sub
```

Comment:
we still don't have decision regarding subs.cljs . so better to move it there, and use reg-root-subs